fuse_msgs/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>fuse_msgs</name>
+  <version>0.0.0</version>
```

Comment:
Remember to sync the version with the rest of the packages, so you don't have issues when you release a new version of the repo.